CefSharp/Internals/TaskExtensions.cs

```diff
@@ -89,8 +89,14 @@ public static TaskCompletionSource<TResult> WithTimeout<TResult>(this TaskComple
         }
 
         /// <summary>
+        /// <para>
```

Comment:
If we go with your proposal, can this method be deleted?

Reply:
I guess it could, it's just still used in a few places in the examples.

Reply:
We can look at upgrading the example projects to .Net 4.6 if required. 

Only the core projects need to remain as .Net 4.5.2

Reply:
I'll let you do that then.
I also just noticed it was used in two places in the `CefSharp.Core` project, and I'm much less familiar with C++/CLI to handle that case.